pallets/pp/src/lib.rs

```diff
@@ -144,7 +144,12 @@ fn send_p2pk_tx<T: Config>(
     ensure!(fund_info.funds >= value, "Caller doesn't have enough funds");
     let outpoints = fund_info.utxos.iter().map(|x| x.0).collect::<Vec<H256>>();
 
-    T::Utxo::send_conscrit_p2pk(caller, dest, value, &outpoints)
+    T::Utxo::send_conscrit_p2pk(caller, dest, value, &outpoints).map(|_| {
```

Comment:
This is a bit nitpicky but conscrit is an annoying "word"

Reply:
I'll think of something else

Reply:
It's not all that important to me. I'm happy to move forwards with this name even if it annoys me. I haven't had chance to do anything but a brief skim so far. I'll look through your pr properly later

Reply:
Truth be told, I don't like and I think Amin doesn't like it either but `send_consensus_critical_p2pk` was too verbose in my opinion

Reply:
Yeah I agree it's a bit long. Let's leave it for now. If someone has a good idea before this is approved let's change it if not then I guess we deal with it. If nobody has a better idea there's not much we can do

Reply:
I renamed them to `submit_c2pk_tx()` and `submit_c2c_tx()`. The consensus-criticalness is implied